threadservice/threadservice.go

```diff
@@ -41,14 +39,14 @@ type Threadservice interface {
 	// AddFollower to a thread.
 	AddFollower(ctx context.Context, id thread.ID, pid peer.ID) error
 
-	// AddRecord with body. See AddOption for more.
-	AddRecord(ctx context.Context, body format.Node, opts ...AddOption) (Record, error)
+	// AddRecord with body.
+	AddRecord(ctx context.Context, id thread.ID, body format.Node) (Record, error)
```

Comment:
No more `opts` for pushing the new record to other addrs?

Reply:
Nope, that was part of the p2p invite mechanism that got thrown out in favor of links.